include/Graphs/SVFGNode.h

```diff
@@ -429,6 +429,43 @@ class InterMSSAPHISVFGNode : public MSSAPHISVFGNode
     const CallBlockNode* callInst;
 };
 
+/*
+ * Dummy node which encodes propagation of an object/version pair.
+ */
+class DummyVersionPropSVFGNode : public VFGNode
```

Comment:
Can we make sure that the users won't use this type of node and it can only be used in VSFS?  Just wondering any way we can make it as a subtype of `MPhil` or `MIntraPhi`?

Reply:
It can be, technically. Though there might be a bit of confusion, for example, what would fields like `cpts`, `res`, and `opVers` be? They'll just sit uninitialised. We'll also have to add our own fields which are irrelevant to `MPhi`/`MIntraPhi` nodes.

Reply:
Correct. It might not be good to subclass them.